modules/ROOT/pages/_partials/block-caveats.adoc

```diff
@@ -1,7 +1,7 @@
 
 
 // tag::icr-community-redirect[]
-Unless otherwise stated this content relates only to replications run using the Inter-Sync Gateway Replication (v2) protocol in {enterprise} deployments.
+Unless otherwise stated this content relates only to Inter-Sync Gateway replications in {enterprise} deployments.
```

Comment:
I do not know the context of this statement and page. Sounds confusing.

In general community is a strict subset of enterprise features. And we should not be confusing "Enterprise deployments" with "Enterprise Edition" .... Community edition can be deployed by Enterprises.

Reply:
{enterprise} expands as Enterprise Edition with a link to the comparison page https://www.couchbase.com/products/editions